File: Python/card.py
from payment import Payment


class Card(Payment):
    numberCard = int
    dateCard = str
    cvv = str


    def __init__(self, id, numberCard, dateCard, cvv) -> None:
        super().__init__(id)
        self.numberCard = numberCard
        self.dateCard = dateCard
        self.cvv = cvv

File: PHP/Card.php
<?php

require_once('./Payment.php');

class Card extends Payment {
    public $numberCard;
    public $dateCard;
    public $cvv

    public function __construct($id, $numberCard, $dateCard, $cvv) {
        parent::__construct($id);
        $this->numberCard = $numberCard;
        $this->dateCard = $dateCard;
        $this->cvv = $cvv;
    }
}

?>